### PR TITLE
feat: change my email and about

### PR DESCRIPTION
**Pull request by claby2**

### CODE REVIEW

File: src/pages/about.astro [hunk 1]
```diff
@@ -27,9 +27,9 @@ const people = [
         name: "Edward Wibowo",
```
Comment (virchau13) [inline src/pages/about.astro:27]: are you sure

Comment (claby2) [reply src/pages/about.astro:27]: uhhh

**Review by priime0 — APPROVED** [review]
lgtm

**Comment by smjleo** [comment]
😡😡😡

**Comment by claby2** [comment]
> rageragerage

plz

**Review by smjleo — CHANGES_REQUESTED** [review]
lbtm

**Comment by claby2** [comment]
can someone merge please o.o

**Pull request merged**